pkg/providers/openstack/provider.go

```diff
@@ -464,6 +401,104 @@ func (p *Provider) imagePackages(image *images.Image) *types.ImagePackages {
 	return &result
 }
 
+func (p *Provider) isImageBelongsToOrganization(image *images.Image, organizationID string) bool {
+	value := image.Properties["unikorn:organization:id"]
+	return value != nil && value != organizationID
```

Comment:
Given how this is being used (removing images private to this organization), I'd prefer something like `imagePublicOrOwnedByOrganization()`.  This logic would then be:

`return value == nil || value == organizationID`

Which is easier to understand I think 😄  Especially seeing as the function name and logic are at odds with one another!